src/foundation.cs

```diff
@@ -16086,7 +16086,168 @@ interface NSMeasurementFormatter : NSSecureCoding {
 		string ToString (NSUnit unit);
 	}
 
-	[Watch (2,0), TV (9,0)]
+	[Mac (10, 8)][iOS (6, 0)][Watch (2, 0)][TV (9, 0)]
+	[BaseType (typeof (NSObject), Name = "NSXPCConnection")]
+	[DisableDefaultCtor]
+	interface NSXpcConnection
+	{
+		[Export ("initWithServiceName:")]
+		[NoiOS][NoWatch][NoTV]
+		IntPtr Constructor (string xpcServiceName);
+
+		[Export ("serviceName")]
+		string ServiceName { get; }
+
+		[Export ("initWithMachServiceName:")]
+		[NoiOS][NoWatch][NoTV]
+		[Internal] // See FIXME in Foundation/NSXpcConnection.cs
+		IntPtr Constructor (string machServiceName, nint options);
```

Comment:
So, my question regarding this and the comment in NSXpcConnection.cs. Does it mean that you are trying to pass the enum to the constructor? You probably can do what you want using the BindAssAttribute on the parameter to go from nint to the enum. Else, you can always use the WrapAttribute to to the conversion.